python/src/phylanx.cpp

```diff
@@ -551,7 +551,24 @@ PYBIND11_MODULE(_phylanx, m)
                     phylanx::ir::node_data<double>{d})};
             });
         },
-        "create a new variable from a floating point value");
+        "create a new variable from a vector floating point values");
+    execution_tree.def("var",
+        [](const std::vector<std::vector<double>>& d) {
+            std::size_t nx = d.size();
+            std::size_t ny = d[0].size();
+            blaze::DynamicMatrix<double> m{nx, ny};
+            for(int i=0;i<nx;i++) {
+              for(int j=0;j<ny;j++) {
+                m(i,j) = d[i][j];
+              }
+            }
+            return hpx::threads::run_as_hpx_thread([&]() {
```

Comment:
If this was turned into a constructor for `ir::node_data<>` the conversion would be automatic.